src/allmydata/storage/http_client.py

```diff
@@ -36,18 +37,69 @@ def __init__(self, code, *additional_args):
         self.code = code
 
 
-def _decode_cbor(response):
+# Schemas for server responses.
+#
+# Tags are of the form #6.nnn, where the number is documented at
+# https://www.iana.org/assignments/cbor-tags/cbor-tags.xhtml. Notably, #6.258
```

Comment:
Aha, _there's_ the reference for 6.258.  I guess it'd be nice to duplicate it in the spec doc.